akka-docs/src/main/paradox/typed/actor-lifecycle.md

```diff
@@ -129,11 +129,11 @@ but sometimes you might want to spawn new actors from the outside of the guardia
 per HTTP request.
 
 That is not difficult to implement in your behavior, but since this is a common pattern there is a predefined
-message protocol and implementation of a behavior for this. It can be used as the guardian actor of the `ActorSystem`,
-possibly combined with `Behaviors.setup` to start some initial tasks or actors. Child actors can then be started from
-the outside by `tell`ing or `ask`ing `SpawnProtocol.Spawn` to the actor reference of the system. Using `ask` is
+message protocol and implementation of a behavior for this. It can be used as the guardian actor of the @apidoc[ActorSystem](typed.ActorSystem),
+possibly combined with @apidoc[Behaviors.setup](typed.*.Behaviors$) {scala="#setup[T](factory:akka.actor.typed.scaladsl.ActorContext[T]=%3Eakka.actor.typed.Behavior[T]):akka.actor.typed.Behavior[T]" java="#setup(akka.japi.function.Function)"} to start some initial tasks or actors. Child actors can then be started from
+the outside by @apidoc[tell](typed.ActorRef) {scala="#tell(msg:T):Unit" java="#tell(T)"}ing or `ask`ing @apidoc[SpawnProtocol.Spawn] to the actor reference of the system. Using `ask` is
```

Comment:
I couldn't find the method `ask`. Which should I take? I need help :)

Reply:
`akka.actor.typed.scaladsl.AskPattern.Askable#ask` and `akka.actor.typed.javadsl.AskPattern#ask`